crates/sallyport/src/item/enarxcall/mod.rs

```diff
@@ -60,29 +63,35 @@ pub enum Number {
     /// SGX quote size request call number.
     GetSgxQuoteSize = 0x05,
 
+    /// SGX CRLs request
```

Comment:
If we are reordering these, let's come up with a convention and group things close?

For example, we could use:
- 0x00 - 0x0F for all SGX calls
- 0x10 - 0x1F for all SNP calls
- 0x20 - 0x9F reserved for future technologies
- 0xA0 - 0xFF for all other calls

Reply:
That would be good. I also noticed that the numbering went from `0x08`, `0x09`, ... `0x10`, `0x11`, so I fixed that.